packages/docs/PropTables/ModalPropTable.tsx

```diff
@@ -19,24 +16,23 @@ export const ModalPropTable: React.FC = () => (
     <PropTable.Prop name="closeOnEscKey" types="boolean" defaults="true">
       Controls whether <Code>onClose</Code> is called when pressing the ESC key.
     </PropTable.Prop>
+    <PropTable.Prop name="header" types="string">
+      Header title of the modal.
```

Comment:
🤷‍♀️Maybe different verbiage, but along these lines?
```suggestion
      Sets visible text that describes the content of the modal.
```